contributing/issue-priorities.md

```diff
@@ -15,18 +15,18 @@ limitations under the License.
 -->
 
 ## AMP GitHub Issue Priorities
-The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
+The AMP team is using the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
 These new guidelines can give you also a good overview of when to expect updates or closure of issues.
 
-In order to add a priority to your issues use our [list of GitHub labels](https://github.com/ampproject/amphtml/labels).
+In order to add a priority to your issues use one from the [list of GitHub labels](https://github.com/ampproject/amphtml/labels). Do expect the issue priority may change depending on the team's roadmap and backlog. 
```

Comment:
Do we want to make it completely optional for the filer to set a priority?  In that case, I think we can phrase this as:

"If you would like to give a hint regarding the priority of the issue you can add the appropriate priority [GitHub label](https://github.com/ampproject/amphtml/labels).  Note that the priority may be changed as the issue is triaged based on the project's roadmap and backlog."